### PR TITLE
`hash` module setup

### PR DESCRIPTION
**Pull request by ilitteri**

### CODE REVIEW

**Review by juanbono — APPROVED** [review]

**Review by MauroToscano — APPROVED** [review]

**Review by MauroToscano — APPROVED** [review]
Ok

File: crypto/src/hash/traits.rs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+use lambdaworks_math::field;
+
+/// Interface to Collision Resistant Hashes.
+pub trait CryptoHash<F>
```
Comment (schouhy) [inline crypto/src/hash/traits.rs:4]: We are following a name convention for traits to be very distinguishable from the structs that implement them. Something like `IsXXX`  or `HasXXX` for traits or even adjectives if possible. Then the struct names would be nouns. I would name this `IsCryptoHash`. What do you think?

Comment (ilitteri) [reply crypto/src/hash/traits.rs:4]: Thanks for pointing that out. I'm updating the name.

**Review by schouhy — APPROVED** [review]
Looks good! Thanks!

**Pull request merged**